orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/pipeline/PreconfiguredWebhookStage.groovy

```diff
@@ -43,6 +55,24 @@ class PreconfiguredWebhookStage extends WebhookStage {
       throw new PreconfiguredWebhookNotFoundException((String) stage.type)
     }
 
+    if (preconfiguredWebhook.permissions) {
+      String user = AuthenticatedRequest.getSpinnakerUser().orElse("anonymous")
+      def userPermission = fiatService.getUserPermission(user)
+
+      def isAllowed = preconfiguredWebhook.isAllowed("WRITE", userPermission.roles)
+      if (!isAllowed) {
+        throw new PreconfiguredWebhookUnauthorizedException((String) user, (String) stage.type)
+      }
+    }
+
+    def user = new User()
+    if (stage.lastModified) {
+      user.setAllowedAccounts(stage.lastModified.allowedAccounts)
+      user.setUsername(stage.lastModified.user)
+
+      UserPermission.View userPermission = fiatService.getUserPermission(user.toString())
```

Comment:
This doesn't seem to be used to check the preconfigured webhook

Reply:
Sorry, I left it here from an old test, removed now